<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="138042" started-at="2019-06-26T19:04:12Z" finished-at="2019-06-26T19:06:30Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="138042" started-at="2019-06-26T19:04:12Z" finished-at="2019-06-26T19:06:30Z">
      <class name="com.saal.qa.testcases.A_Scenario1PageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.saal.qa.testcases.A_Scenario1PageTest@4501b7af]" name="setUp" is-config="true" duration-ms="53603" started-at="2019-06-26T23:04:13Z" finished-at="2019-06-26T23:05:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="validateCreateNewContact(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.saal.qa.testcases.A_Scenario1PageTest@4501b7af]" name="validateCreateNewContact" duration-ms="83191" started-at="2019-06-26T23:05:07Z" description="Validate Scenario 1" data-provider="getBookingTestData" finished-at="2019-06-26T23:06:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[DXB]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[TRV]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[2019-07-01]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementListHandler.invoke(LocatingElementListHandler.java:39)
	at com.sun.proxy.$Proxy12.get(Unknown Source)
	at com.saal.qa.pages.A_Scenario1Page.bookingticket(A_Scenario1Page.java:174)
	at com.saal.qa.testcases.A_Scenario1PageTest.validateCreateNewContact(A_Scenario1PageTest.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateCreateNewContact -->
      </class> <!-- com.saal.qa.testcases.A_Scenario1PageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
